common.js

```diff
@@ -293,8 +305,7 @@ module.exports = {
     // appPath is predictable (e.g. before .app is renamed for mac)
     if (opts.prune || opts.prune === undefined) {
       operations.push(function (cb) {
-        debug('Running npm prune --production')
```

Comment:
This should come back in some form or another. We need to be able to see when the prune operation occurs and with what package manager. It's pretty helpful for debugging user problems.